app/frontend/src/features/communities/events/EventCard.tsx

```diff
@@ -95,60 +112,42 @@ export default function EventCard({ event, className }: EventCardProps) {
         <CardMedia
           src={event.photoUrl || eventImagePlaceholder}
           className={classes.image}
-          component="img"
-        />
-        <CardContent>
-          {eventCreator ? (
-            <Typography
-              variant="caption"
-              component="p"
-              className={classes.subtitle}
-              noWrap
-            >
-              {getByCreator(eventCreator.name)}
-            </Typography>
-          ) : (
-            <Skeleton width={100} />
+        >
+          {event.onlineInformation && (
+            <Chip className={classes.chip} size="medium" label={ONLINE} />
           )}
+        </CardMedia>
+        <CardContent>
           <LinesEllipsis
             maxLine={2}
             text={event.title}
             component="h3"
             className={classes.title}
           />
+          <Typography className={classes.subtitle} variant="body2">
+            {event.offlineInformation
+              ? event.offlineInformation.address
+              : VIEW_DETAILS_FOR_LINK}
+          </Typography>
           <ul className={classes.detailsList}>
             <li>
               <CalendarIcon className={classes.icon} />
-              <Typography variant="body2" noWrap>
-                {dayjs(startTime).format("LL")}
-              </Typography>
-            </li>
-            <li>
-              <ClockIcon className={classes.icon} />
-              <Typography variant="body2" noWrap>
-                {`${dayjs(startTime).format("LT")} - ${dayjs(endTime).format(
-                  "LT"
+              <Typography variant="body1" noWrap>
+                {`${startTime.format("LLL")} - ${endTime.format(
+                  endTime.diff(startTime, "day") >= 1 ? "LLL" : "LT"
```

Comment:
Logic is not correct, 11pm to 1am the next day is 0 days difference.
```suggestion
                  endTime.isSame(startTime, "day") ? "LT" : "LLL"
```

Reply:
Ahh nice catch, I misread this in the doc and didn't realise you can also pass in a unit in these comparison functions 👍